chia/farmer/farmer_api.py

```diff
@@ -305,8 +308,10 @@ async def respond_signatures(self, response: harvester_protocol.RespondSignature
         if response.sp_hash not in self.farmer.sps:
             self.farmer.log.warning(f"Do not have challenge hash {response.challenge_hash}")
             return None
+
```

Comment:
```suggestion
```

nit, feel free to just close this out but i still want to point to it because its unrelated here.